data/src/main/java/com/globant/data/database/HarryPotterDataBaseImpl.kt

```diff
@@ -2,22 +2,26 @@ package com.globant.data.database
 
 import androidx.room.Database
 import androidx.room.RoomDatabase
+import com.globant.data.entity.HouseDetailEntity
 import com.globant.data.entity.HouseRoom
 import com.globant.data.mapper.SpellDataBaseMapper
-import com.globant.domain.database.HarryPotterRoomDataBase
+import com.globant.domain.database.HarryPotterDataBase
 import com.globant.domain.entity.Spell
 import com.globant.data.entity.SpellRoom
 import com.globant.data.mapper.HouseDataBaseMapper
+import com.globant.data.mapper.HouseDetailDatabaseMapper
 import com.globant.domain.entity.House
+import com.globant.domain.entity.HouseDetail
 import com.globant.domain.util.Result
 import java.lang.Exception
 
-@Database(entities = [SpellRoom::class, HouseRoom::class], version = 1)
-abstract class HarryPotterRoomDataBaseImpl : RoomDatabase(), HarryPotterRoomDataBase {
+@Database(entities = [SpellRoom::class, HouseRoom::class, HouseDetailEntity::class], version = 2)
```

Comment:
why did you upgrade the version?

Reply:
I don´t need to do this, my bad! Refactored!